src/velocity_moments.jl

```diff
@@ -298,19 +307,22 @@ function integrate_over_negative_vpa(integrand, dzdt, vpa_wgts, wgts_mod)
     end
     return vpa_integral
 end
-function enforce_moment_constraints!(fvec_new, fvec_old, vpa, z, composition, moments)
+function enforce_moment_constraints!(fvec_new, fvec_old, vpa, z, r, composition, moments, dummy_sr)
     #global @. dens_hist += fvec_old.density
     #global n_hist += 1
 
     # pre-calculate avgdens_ratio so that we don't read fvec_new.density[:,is] on every
     # process in the next loop - that would be an error because different processes
     # write to fvec_new.density[:,is]
-    # This loop needs to be @s_z_loop_s because it fills the (not-shared)
-    # composition.scratch buffer to be used within the @s_z_loop_s below, so the values
+    # This loop needs to be @s_r_z_loop_s because it fills the (not-shared)
+    # dummy_sr buffer to be used within the @s_z_loop_s below, so the values
```

Comment:
```suggestion
    # This loop needs to be @loop_s_r because it fills the (not-shared)
    # dummy_sr buffer to be used within the @loop_s_r below, so the values
```